check-ostree.yaml

```diff
@@ -1019,7 +1020,85 @@
       when:
         - (ansible_facts['distribution'] == 'RedHat' and ansible_facts ['distribution_version'] is version('9.2', '>=')) or
           ((ansible_facts['distribution'] == 'RedHat' and ansible_facts ['distribution_version'] is version('8.8', '>=')) and ansible_facts ['distribution_version'] is version('9.0', '!=') and ansible_facts ['distribution_version'] is version('9.1', '!=')) or
-          (ansible_facts['distribution'] == 'CentOS')
+          (ansible_facts['distribution'] == 'CentOS' and ansible_facts ['distribution_version'] is version('8', '!='))
```

Comment:
Could you please remove this line of code? Thanks.

Reply:
Removed.